rtengine/procparams.cc

```diff
@@ -129,22 +131,22 @@ RetinexParams::RetinexParams ()
     setDefaults ();
 }
 
-void RetinexParams::getDefaulttransmissionCurve(std::vector<double> &curve)
+void RetinexParams::getDefaulttransmissionCurve (std::vector<double> &curve)
```

Comment:
@Desmis Jaques, is there something wrong with your editor setup? Because it changes whitespace in places, you didn't touch. It's not that this is so important, but makes the review a bit harder. And while whitespace is changed in places uncalled-for, the lines below (`double v[12] = ...`) would definitely need some whitespace massage. 😃